buddyboss/buddyboss-platform
# PROD-665 #3226: Woocommerce product edit page postbox expand/collapse button issue fixed

### All Submissions:






### Changes proposed in this Pull Request:



Fixes #3226  .

### How to test the changes in this Pull Request:

1. Go to Woocommerce Product edit page
2. Click on the any postbox expand/collapse button from right panel
3. See no error

### Proof Screenshots or Video

https://share.vidyard.com/watch/oLxruBJXA6cz1CavpSzfe1?

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
